### ISSUE TITLE
iruby and rbenv

### ISSUE THREAD

**Issue by agisga** [SciRuby/iruby#36]
I use Ruby 2.2.2 installed via [rbenv](https://github.com/sstephenson/rbenv) on Ubuntu 15.04.
IRuby installed without problems, but when I tried to run `iruby notebook` it couldn't load any gems, giving error messages like:

```
LoadError: cannot load such file -- some_gem_i_have_installed
/usr/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
...
```

After spending an hour, trying to make it work, I ended up switching to system Ruby via `rbenv shell system`, and installing all my gems into system Ruby. After that everything worked fine.

Is there a way to make iruby work with rbenv Ruby without reinstalling all the gems into system Ruby?

**Comment by v0dro** [comment]
See #31

**Comment by agisga** [comment]
Possibly, although I cannot see a clear connection. I have no problem running iruby gem from any directory, but iruby has trouble finding my installed gems.

**Comment by v0dro** [comment]
Can you try including the gems in a Gemfile and see if it works?

**Comment by agisga** [comment]
Can confirm that the problem is solved with [45fd71b](https://github.com/SciRuby/iruby/commit/45fd71b046fada03f42570107443e1d18bc78006) after running `iruby register --force`.

**Comment by minad** [comment]
:)
